### PR TITLE
remove more hard coding of per month

### PR DESCRIPTION
**Pull request by andymckay**
- there's more to do on localisation of emails, like all of it, but this is a start and makes per month depend upon the recurrence setting
- related to mozilla/payments-ui#348

### CODE REVIEW

**Comment by tofumatt** [comment]
I'll say this is related to https://github.com/mozilla/payments-ui/issues/296, in a roundabout way, so marking as such so I remember to look at this as well when I get around to fixing it :smile:

**Comment by andymckay** [comment]
Agreed. Although really all of this should be nuked for something more cunning like: mozilla/payments#111

File: payments_service/braintree/templates/braintree/emails/subscription_canceled.html [hunk 1]
```diff
@@ -9,7 +9,7 @@
         </tr>
         <tr class="desc">
             <th scope="row">{{ product.description }}</th>
-            <td>${{ moz_trans.amount }} <span>per month</span></td>
+            <td>{{ recurring_amount }}</td>
```
Comment (kumar303) [inline payments_service/braintree/templates/braintree/emails/subscription_canceled.html:12]: `amount_and_terms` ? I dunno, 'recurring' won't always be true so this seems odd. But, meh, names.

**Comment by kumar303** [comment]
r+wc

**Pull request merged**